.github/workflows/populate-cache.yml

```diff
@@ -0,0 +1,34 @@
+# A dummy workflow that populates the runner with dependencies
+# and pops them in the actions cache, so we know there is
+# always a fallback for subsequent PRs
+#
+name: Populate Runner Cache
+on:
+  push:
+    branches:
+      - main
+      - feature/**
+  pull_request:
```

Comment:
[question]  Is there a reason we want this to run on any PR rather than just main (or feature) branch?

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/695/commits/66cadf16285127b2ced2c42c58fab9e6bd27b48a#r1009488490